### ISSUE TITLE
"Connection aborted" error when running icloudpd

### ISSUE THREAD

**Issue by alexodee** [icloud-photos-downloader/icloud_photos_downloader#676]
When I attempt to run the icloudpd command with the --username, --password, -n, and --list options, I encounter a "Connection aborted" error, which appears to be stemming from a "ConnectionResetError".

**Steps to reproduce:**

1. I executed the following command: icloudpd --username [my_username] --password [my_password] -n --list (note: I've replaced my actual username and password with placeholders here for security reasons).
2. I immediately received a full error traceback, which I've attached below.

**Error traceback:**

Traceback (most recent call last):
  File "urllib3\connectionpool.py", line 714, in urlopen
  File "urllib3\connectionpool.py", line 403, in _make_request
  File "urllib3\connectionpool.py", line 1053, in _validate_conn
  File "urllib3\connection.py", line 419, in connect
  File "urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
  File "urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
  File "ssl.py", line 517, in wrap_socket
  File "ssl.py", line 1075, in _create
  File "ssl.py", line 1346, in do_handshake
ConnectionResetError: [WinError 10054] Une connexion existante a dû être fermée par l’hôte distant

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "requests\adapters.py", line 486, in send
  File "urllib3\connectionpool.py", line 798, in urlopen
  File "urllib3\util\retry.py", line 550, in increment
  File "urllib3\packages\six.py", line 769, in reraise
  File "urllib3\connectionpool.py", line 714, in urlopen
  File "urllib3\connectionpool.py", line 403, in _make_request
  File "urllib3\connectionpool.py", line 1053, in _validate_conn
  File "urllib3\connection.py", line 419, in connect
  File "urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
  File "urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
  File "ssl.py", line 517, in wrap_socket
  File "ssl.py", line 1075, in _create
  File "ssl.py", line 1346, in do_handshake
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "starters\icloud.py", line 5, in <module>
  File "pyicloud_ipd\cmdline.py", line 209, in main
  File "pyicloud_ipd\base.py", line 204, in __init__
  File "pyicloud_ipd\base.py", line 220, in authenticate
  File "requests\sessions.py", line 637, in post
  File "pyicloud_ipd\base.py", line 71, in request
  File "requests\sessions.py", line 589, in request
  File "requests\sessions.py", line 703, in send
  File "requests\adapters.py", line 501, in send
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))
[22320] Failed to execute script 'icloud' due to unhandled exception!

**Environment:**

    icloudpd version: 1.15.1
    Operating System: Windows 11 22H2

**Additional Information:**

I've already tried disabling my firewall to see if it was causing the issue, but the error persists. Any additional details or steps that might help in resolving this issue would be greatly appreciated.

**Comment by alexodee** [comment]
I did a test with another pc and it worked. Can you help me unblock on my main computer?
Main computer: Windows 11 22H2, Kaspersky Internet Security
Other computer: Windows 10 22H2, Avast
Both computers are on the same network.
Thank you for your help

**Comment by AndreyNikiforov** [comment]
Please run `icloudpd --help` to get list of valid parameters. Both `-n` and `--list` are invalid parameters.

Also, make sure you are running correct script: `[22320] Failed to execute script 'icloud' due to unhandled exception!` is your log suggests that you ran `icloud`, not `icloudpd`.

**Comment by alexodee** [comment]
Thanks for your answer. I had used the icloud script and not icloudpd. I've now used icloudpd and I get the same error (which I still don't get on another pc):

**icloudpd --directory [directory] --username [email] --password [password]**
2023-08-26 12:36:23 DEBUG    Authenticating...
Traceback (most recent call last):
  File "urllib3\connectionpool.py", line 714, in urlopen
  File "urllib3\connectionpool.py", line 403, in _make_request
  File "urllib3\connectionpool.py", line 1053, in _validate_conn
  File "urllib3\connection.py", line 419, in connect
  File "urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
  File "urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
  File "ssl.py", line 517, in wrap_socket
  File "ssl.py", line 1075, in _create
  File "ssl.py", line 1346, in do_handshake
ConnectionResetError: [WinError 10054] Une connexion existante a dû être fermée par l’hôte distant

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "requests\adapters.py", line 486, in send
  File "urllib3\connectionpool.py", line 798, in urlopen
  File "urllib3\util\retry.py", line 550, in increment
  File "urllib3\packages\six.py", line 769, in reraise
  File "urllib3\connectionpool.py", line 714, in urlopen
  File "urllib3\connectionpool.py", line 403, in _make_request
  File "urllib3\connectionpool.py", line 1053, in _validate_conn
  File "urllib3\connection.py", line 419, in connect
  File "urllib3\util\ssl_.py", line 449, in ssl_wrap_socket
  File "urllib3\util\ssl_.py", line 493, in _ssl_wrap_socket_impl
  File "ssl.py", line 517, in wrap_socket
  File "ssl.py", line 1075, in _create
  File "ssl.py", line 1346, in do_handshake
urllib3.exceptions.ProtocolError: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "starters\icloudpd.py", line 5, in <module>
  File "click\core.py", line 1157, in __call__
  File "click\core.py", line 1078, in main
  File "click\core.py", line 1434, in invoke
  File "click\core.py", line 783, in invoke
  File "icloudpd\base.py", line 299, in main
  File "icloudpd\base.py", line 721, in core
  File "icloudpd\authentication.py", line 30, in authenticate_
  File "pyicloud_ipd\base.py", line 204, in __init__
  File "pyicloud_ipd\base.py", line 220, in authenticate
  File "requests\sessions.py", line 637, in post
  File "pyicloud_ipd\base.py", line 71, in request
  File "requests\sessions.py", line 589, in request
  File "requests\sessions.py", line 703, in send
  File "requests\adapters.py", line 501, in send
requests.exceptions.ConnectionError: ('Connection aborted.', ConnectionResetError(10054, 'Une connexion existante a dû être fermée par l’hôte distant', None, 10054, None))
[864] Failed to execute script 'icloudpd' due to unhandled exception!

Thanks for your help
